src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewAboutToSubmitHandler.java

```diff
@@ -37,17 +39,26 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
                                                           String userAuthorisation) {
         SscsCaseData caseData = callback.getCaseDetails().getCaseData();
 
-        PreSubmitCallbackResponse<SscsCaseData> response = new PreSubmitCallbackResponse<>(caseData);
-
         PostHearing postHearing = caseData.getPostHearing();
         log.info("Review Post Hearing App: handling action {} for case {}", postHearing.getReviewType(), caseData.getCcdCaseId());
 
-        if (response.getErrors().isEmpty()) {
```

Comment:
Do we still want to check response for errors first here? Why not?

Reply:
there's no possibility of having an error before this